protos/flyteidl/admin/version.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package flyteidl.admin;
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin";
+
+// Represents a request structure to get version of flyteadmin.
+message Version {
+    // Specifies the GIT sha of the build
+    string	Build = 1;
+
+    // Version for the build, should follow a semver
```

Comment:
I wonder if it be worthwhile to add a proper semver message or keep with the parsable string convention?

Reply:
@katrogan this is the auto-injected version into the binary, everytime anyone builds it. this is better than just semver, as someone may be running their own version